content/docs/start/data-pipelines.md

```diff
@@ -24,8 +24,8 @@ https://youtu.be/71IGzyH95UY
 ## Pipeline stages
 
 Use `dvc run` to create _stages_. These represent processes (source code tracked
-with Git) that form the steps of a pipeline. Stages also connect code to its
-data input and output. Let's transform a Python script into a
+with Git) which form the steps of a _pipeline_. Stages also connect code to its
+corresponding data _input_ and _output_. Let's transform a Python script into a
```

Comment:
💅 Could be "theirs" (stages) or "its" (code). What about "the"
```suggestion
with Git) which form the steps of a _pipeline_. Stages also connect code to the
corresponding data _input_ and _output_. Let's transform a Python script into a
```

Reply:
I believe its code, not stages, so "its."